forge-gui/res/cardsfolder/upcoming/perforator_crocodile.txt

```diff
@@ -0,0 +1,11 @@
+Name:Perforator Crocodile
+ManaCost:3 B B
+Types:Creature Crocodile
+PT:6/5
+K:Cycling:1 B
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigRepeat | TriggerDescription$ When CARDNAME enters the battlefield, for each creature your opponents control, conjure a card named Stab Wound onto the battlefield attached to that creature.
+SVar:TrigRepeat:DB$ RepeatEach | RepeatCards$ Creature.OppCtrl | RepeatSubAbility$ DBConjure
```

Comment:
This causes a crash 
![image](https://github.com/Card-Forge/forge/assets/67333662/022d7d65-831a-4506-b820-7ee3c3335ad5)

Any thoughts why @Northmoc

Reply:
the LastState params in MakeCard should be obtained from Game